Ironmon-Tracker.lua

```diff
@@ -2,7 +2,7 @@
 -- Created by besteon, based on the PokemonBizhawkLua project by MKDasher
 
 -- The latest version of the tracker. Should be updated with each PR.
-TRACKER_VERSION = "0.2.11"
+TRACKER_VERSION = "0.2.2"
```

Comment:
Consider [Semantic Versioning](https://semver.org/) where each component is a separate number, as opposed to some hybrid decimal/fractional number. Therefore the "12th patch" (0.2.12) is next, and the "2nd patch" would come before the 11th.

Reply:
Crap, I had written a bunch of stuff here, clicked "Resolve" and it didn't leave the comment... I'm too used to Azure DevOps. 🤦‍♂️

Anyway, what I had said previously was that the last version should actually have been 0.2.2 and this would then be 0.2.3, if we follow semantic versioning. Calling it 0.2.11 was my mistake as I was trying to say it was just a very minor update to 0.2.1...

I think we can continue on with semantic versioning from here, but I'll leave this as 0.2.2 for now.